### ISSUE TITLE
Nmap Fails to Build

### ISSUE THREAD

**Issue by godylockz** [nmap/nmap#2556]
** Bug **
NMAP fails to build/install due to `nmap-payloads` being deleted in:
https://github.com/nmap/nmap/commit/2fcfac93292f65a7ca67294d48061fc45e1b323f

Recommend removing Line 332 in https://github.com/nmap/nmap/blob/master/Makefile.in
And any other references to `nmap-payloads`.
```sh
$(INSTALL) -c -m 644 nmap-payloads $(DESTDIR)$(nmapdatadir)/
```

Should be a quick issue. 

Thanks!

**Comment by benpratt** [comment]
Appears to be a duplicate of #2553.

A pull-request (#2554) was submitted 11 days ago. Once it is merged it should resolve this issue.

**Comment by godylockz** [comment]
Fixed in https://github.com/nmap/nmap/commit/70dc5434f41109cd0e9000f9082cbe3b5a6a9d31
